gapic/utils/options.py

```diff
@@ -41,20 +41,22 @@ class Options:
     lazy_import: bool = False
     old_naming: bool = False
     add_iam_methods: bool = False
+    metadata: bool = False
     # TODO(yon-mg): should there be an enum for transport type?
     transport: List[str] = dataclasses.field(default_factory=lambda: [])
 
     # Class constants
     PYTHON_GAPIC_PREFIX: str = 'python-gapic-'
     OPT_FLAGS: FrozenSet[str] = frozenset((
+        'add-iam-methods',      # microgenerator implementation for `reroute_to_grpc_interface`
+        'lazy-import',          # requires >= 3.7
+        'metadata',             # generate gapic metadata JSON file
```

Comment:
totally insignificant nit: in mixed-case comments, we can capitalize GAPIC, since it's an acronym

Reply:
Fixed